SocialFilter/sections/specific.txt

```diff
@@ -73,6 +73,7 @@ tsn.ua##.c-nav__list--dense
 crictime.com##td[height="46"][align="center"]
 livehindustan.com##.socialShare-btn
 gsk.com##.module-social
+avforums.com##div[data-widget-key="forum_overview_share_page"]
```

Comment:
It works, but we have a common rule `##div[data-widget-definition="share_page"]`